content/doc/developer/publishing/releasing-cd.adoc

```diff
@@ -2,7 +2,7 @@
 title: Setting up automated plugin release
 layout: developer
 references:
-- url: /jep/229
```

Comment:
This actually works on jenkins.io (not for the preview site or localhost). Though the change is not needed, avoiding redirects saves some time browsing, so :+1:

Reply:
yeah i could see it working on the jenkins.io when i was comparing but i also see how we usually use this in other files 
![image](https://github.com/jenkins-infra/jenkins.io/assets/76218033/0f60ff51-9d95-4a3f-8899-e6bcc2942bb6)
so i thought this can be an improvement anyway